### PR TITLE
[DEV APPROVED] - 7929 Expose office website

### PR DESCRIPTION
**Pull request by margOnline**
This pr addresses [user story 7929](https://moneyadviceservice.tpondemand.com/restui/board.aspx?acid=f363bea37e60ac292c07aaf8b50f0da9#page=board/5624876525867731357&appConfig=eyJhY2lkIjoiREI4ODcwRjkxMDNDRTM2NTlBMzhDNTRBRTVBNUU1N0UifQ==&boardPopup=userstory/7929/silent) to show the office website if it is different to the firm's website.

It builds on [rad#401](https://github.com/moneyadviceservice/rad/pull/401) which created the ability for admin users to add a specific website for a firm's office.

### CODE REVIEW

File: spec/lib/mas/office_result_spec.rb [hunk 1]
```diff
@@ -26,6 +27,7 @@
     specify { expect(subject.email_address).to eq('postie@example.com') }
     specify { expect(subject.telephone_number).to eq('020 8595 2346') }
     specify { expect(subject.disabled_access).to eq(true) }
+    specify { expect(subject.website).to eq("http://www.postman.com") }
```
Comment (tomas-stefano) [inline spec/lib/mas/office_result_spec.rb]: single quotes?

File: spec/lib/mas/office_result_spec.rb [hunk 2]
```diff
@@ -10,6 +10,7 @@
       'email_address'    => 'postie@example.com',
       'telephone_number' => '020 8595 2346',
       'disabled_access'  => true,
+      'website'          => "http://www.postman.com",
```
Comment (tomas-stefano) [inline spec/lib/mas/office_result_spec.rb]: Can we use single quotes here too?

**Comment by tomas-stefano** [comment]
Minor comment, but 👍 👍

**Comment by Tr4pSt3R** [comment]
👍

**Pull request merged**